.github/workflows/status-page.yml

```diff
@@ -0,0 +1,32 @@
+name: Update status.amp.dev
+on:
+  issues:
+    types: [opened]
+  issue_comment:
+    types: [edited]
+jobs:
+  status-page:
+    if: contains(github.event.issue.title, '🌸 Cherry-pick request')
+    runs-on: ubuntu-latest
+    environment: status_page
+    steps:
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+          registry-url: https://registry.npmjs.org
+      - uses: actions/checkout@v2
+      - name: Add progress comment to cherry-pick issue for Stable and LTS
+        if: github.event_name == 'issues' && github.event.action == 'opened'
+        run: |
+          npm install
+          node ./build-system/status-page/comment.js ${{ github.event.issue.number }} '${{ github.event.issue.body }}' ${{ github.actor }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.AMPPROJECTBOT }}
+      - name: Sync status page with cherry-pick progress
+        if: github.event_name == 'issue_comment' && github.event.action == 'edited' && contains(github.event.comment.body, '🌸 Cherry-pick Progress 🌸')
+        run: |
+          npm install
```

Comment:
During CI, it's recommended to run `npm ci` over `npm i` since `ci` guarantees a clean install and does not modify either of the package files.